gradio/chat_interface.py

```diff
@@ -214,6 +222,18 @@ def __init__(
                 if self._additional_inputs_in_examples:
                     break
 
+        if flagging_mode is None:
+            flagging_mode = os.getenv("GRADIO_FLAGGING_MODE", "never")
```

Comment:
Consider creating a different environment variable? Otherwise anyone who has this env variable already set will see the flagging. Even more problematic, if someone has this set to "auto" will see their `gr.ChatInterface` break
```suggestion
            flagging_mode = os.getenv("GRADIO_FLAGGING_MODE", "never")
```